adapters/repos/db/lsmkv/compactor_replace.go

```diff
@@ -62,6 +62,8 @@ func (c *compactorReplace) do() error {
 		return fmt.Errorf("init: %w", err)
 	}
 
+	panic("oh oh - how did this panic sneak in here?")
```

Comment:
I guess this is intended only for testing the sentry functionality, right? or is it meant to stay :-P ?

Reply:
Good catch, you're absolutely right ;-)